spec/tasks/db_create_spec.cr

```diff
@@ -3,7 +3,7 @@ require "../spec_helper"
 describe Db::Create do
   it "raises a connection error when unable to connect" do
     Avram.temp_config(database_to_migrate: DatabaseWithIncorrectSettings) do
-      expect_raises(Avram::ConnectionError, /Failed to connect to database/) do
+      expect_raises(Exception, /It looks like Postgres is not running/) do
```

Comment:
For some reason, this has to be `Exception`, which is technically correct, but when using the actual subclass, it fails. No clue why.